core/app.ts

```diff
@@ -49,16 +51,36 @@ function createRouter (): VueRouter {
   })
 }
 
+function createRouterProxy (router: VueRouter): VSFRouter {
```

Comment:
Can you maybe use arrow function to keep consistence with the rest of file?

Reply:
And also you can move this function to the `core/helpers` and import here

Reply:
good point, I've change it